e2e/cypress/integration/collapsed_reply_threads/following_spec.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Please add group test metadata, like as follows, so we can easily run this in group.
```
// Group: @collapsed_reply_threads
```